### DISCUSSION TITLE
¿How can I install GLPI agent in SUSE Linux Enterprise Server 15 SP2?

### DISCUSSION THREAD

**Discussion by Jonathan-R23** [glpi-project/glpi-agent#690] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Hi there, 
I'm facing with a problem when I tried to install glpi agent in SUSE Linux Enterprise Server 15 SP2, basically my steps were:
```bash
sudo wget -c https://github.com/glpi-project/glpi-agent/releases/download/1.9/glpi-agent-1.9-linux-installer.pl
sudo perl glpi-agent-1.9-linux-installer.pl
```

but I got an error
**Not supported linux distribution**
So, after that I decided to play with the options:
```
sudo perl glpi-agent-1.9-linux-installer.pl --verbose --force --distro opensuse
```
Getting this error:
```
Found distro: Unknown Linux distribution
Running on linux distro: %s : opensuse : unknown version...
Installing glpi-agent v1.9...
Trying to install glpi-agent v1.9-1 on %s release (opensuse:unknown version)...
Extracting glpi-agent-1.9-1.noarch.rpm ...
Running: dnf -y install glpi-agent-1.9-1.noarch.rpm
Can't exec "dnf": No such file or directory at glpi-agent-1.9-linux-installer.pl line 787.
Failed to install glpi-agent
```
My operating system is:
```
NAME="SLES"
VERSION="15-SP2"
VERSION_ID="15.2"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP2"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp2"
```
Any idea to overcome this problem?
I will appreciate.

**Comment by g-bougard** [comment]
Hi @Jonathan-R23 

did you try to manually install rpm files ?

I'm not aware of the installation process on SLES and what's the commonly used command to install packages. I'm even not sure our rpm are supported on this distro. But if they work and you provide the correct process I'll modify the installer to support SLES.
